src/neuroconv/tools/nwb_helpers/_configure_backend.py

```diff
@@ -27,18 +107,20 @@ def configure_backend(
     is_ndx_events_installed = is_package_installed(package_name="ndx_events")
     ndx_events = sys.modules.get("ndx_events", None)
 
-    nwbfile_objects = nwbfile.objects
+    dataset_configuration_to_nwbfile_object_ids = _resolve_nwbfile_object_ids(
```

Comment:
I would rather avoid having to call this for the most common neuroconv case. Given that this is an edge case can we have this as an external method that is called in those conditions. I don't think that is worth it to add more complexity to the function for this edge case that can be handled outside it.

Reply:
I'd rather not waterbed this condition up to the converter; a function named `configure_backend` ought to do what it says minus the relatively minor detail of exactly equal Pydantic model structures minus randomly chosen object IDs

Reply:
I think the proposal of #863 decouples backend_configuration from the re-maping one extra step.